Source/Python/application/factories.py

```diff
@@ -45,36 +46,41 @@ def __init__(self, repository_type: str, **kwargs) -> None:
 
 
 class RequestHandlersFactory:
-    _HANDLER_TYPES = {
-        "rest": "rest_handlers",
-    }
-    _location_handler: LocationsRequestHandler
-    _traveler_handler: TravelersRequestHandler
-    _event_handler: EventsRequestHandler
+    _location_handler: LocationsRestRequestHandler
+    _traveler_handler: TravelersRestRequestHandler
+    _event_handler: EventsRestRequestHandler
 
     @property
-    def location_handler(self) -> LocationsRequestHandler:
+    def location_handler(self) -> LocationsRestRequestHandler:
```

Comment:
These getters shouldn't be needed any more, as they are not needed by the flask app directly

Reply:
Removed this class entirely.